static/js/protein/chart.ts

```diff
@@ -219,6 +240,76 @@ function addYLabel(
     .text(labelText);
 }
 
+export function drawTissueLegend(id: string, data: ProteinStrData[]): void {
+  // checks if the data is empty or not
+  if (_.isEmpty(data)) {
+    return;
+  }
+  const svg = d3
+    .select("#tissue-score-legend")
+    .append("svg")
+    .attr("width", GRAPH_WIDTH_XL)
+    .attr("height", GRAPH_HEIGHT_XS);
+  const data0 = d3.keys(TISSUE_LEGEND_DICT);
+  // slicing the dictionary in half to display the legend in two rows
+  const data1 = data0.slice(0, 8);
+  const data2 = data0.slice(8, 15);
+  // creating circles for the first row in legend
+  svg
+    .selectAll("mydots1")
+    .data(data1)
+    .enter()
+    .append("circle")
+    .attr("cx", (d, i) => {
+      return NUM_DATA_POINTS + i * GRAPH_HEIGHT_XS;
```

Comment:
curious why the x coordinate of the dot depends on the graph height? Because shouldn't x coordinate be related to width of the graph? 
Also curious how NUM_DATA_POINTS plays into the calculations

Reply:
I want to use the values of NUM_DATA_POINTS and GRAPH_HEIGHT_XS, that's why I used those constants. Is it okay to define more constants with the same values?

Reply:
yes, that's ok to define more constants with the same values

Reply:
Done!